src/components/cqlLibraryLanding/CqlLibraryLanding.tsx

```diff
@@ -52,6 +52,57 @@ function CqlLibraryLanding() {
   const [selectedCQLLibrary, setSelectedCqlLibrary] =
     useState<CqlLibrary>(null);
 
+  const [numberMyCqlLibraries, setNumberMyCqlLibraries] = useState(0);
+  const [numberAllCqlLibraries, setNumberAllCqlLibraries] = useState(0);
+  useEffect(() => {
+    cqlLibraryServiceApi
+      .fetchCqlLibraries(
```

Comment:
Why can't we use existing retrieveLibraries function ?

Reply:
@RohitKandimalla There is a useEffect that uses retrieveLibraries function, the problem is retrieveLibraries function does setPageProps, which will set the page information. So if I use that, I can get the numbers right, but the list and table will display the same info.

Reply:
I get it, but I was wondering why can't we re-use the same piece code.

Reply:
@RohitKandimalla Because if I use retrieveLibraries twice, the list of My Libraries and the list of All Libraries will be the same, they should be different.

Reply:
the output of retrieveLibraries will be dependent on the input parameters.